### ISSUE TITLE
Homebrew for Linux delivers Darwin binary

### ISSUE THREAD

**Issue by hlascelles** [FairwindsOps/polaris#272]
### Homebrew for Linux delivers Darwin binary

On a Linux machine (Mint 19.3)

```bash
brew install FairwindsOps/tap/polaris
# Notice it is downloading Darwin:
# ==> Downloading https://github.com/FairwindsOps/polaris/releases/download/0.6.0/polaris_0.6.0_Darwin_x86_64.tar.gz
```

The binary will not execute (invalid exec format).

I see you are using `goreleaser` which I imagine should have got this right, so I don't know what's going on....

**Comment by sudermanjr** [comment]
@hlascelles Thanks for reporting this. We will take a look

**Comment by rbren** [comment]
Thanks for the report! This will hopefully be fixed with the 1.0 release (🤞 slated for tomorrow). We've updated goreleaser as well as our config.

**Comment by rbren** [comment]
Confirmed fixed in 1.0.0 - let me know if you have any trouble!

**Comment by hlascelles** [comment]
Confirmed, working great, thanks!
